source/documentors/decisions/0002-choosing-rst.rst

```diff
@@ -0,0 +1,220 @@
+Choosing RST
+############
+
+Status
+******
+
+**Provisional**
+
+A major concern with accepting this decision is whether we can overcome the RST
+and Github barrier to entry by investing in training and proper documentation.
+This decision should be re-evaluated once we've put the relevant quickstarts,
+how-tos and references put through their paces.
+
+Context
+*******
+
+Now that we agree on the general goal of a root documentation site, we need to
+figure out how we will write the words for the site.  Per `OEP-19`_ developer
+documentation is written in RST.  However since the audience for all
+documentation is not developers it is worth re-considering and reviewing the
+alternatives and actively make a decision here rather than passively choose what
+already exists.
+
+Decision
+********
+
+We will use *Sphinx-flavored* RST for the root `docs.openedx.org`_ site and any
```

Comment:
What is Sphinx-flavored RST? It's not detailed anywhere.

Reply:
"RST with Sphinx extensions" would be a more technically accurate term. See [reStructuredText — Sphinx documentation](https://www.sphinx-doc.org/en/master/usage/restructuredtext/index.html)